cmd/syncthing/gui.go

```diff
@@ -231,6 +231,7 @@ func (s *apiService) Serve() {
 	s.listenerMut.Unlock()
 
 	if listener == nil {
+		l.Warnln("API: Aborting because no listener configured")
```

Comment:
Well, if we hit this one there there *is* no web UI, since we failed to start it.

If I'm scanning through a log to see why by web UI doesn't exist, I'll be looking for something more than an info.

Unless I've misunderstood when this can occur..?